mettle/api/api.go

```diff
@@ -72,17 +69,29 @@ func serve(opts grpcutil.Opts, name, requestQueue, responseQueue, preResponseQue
 		preResponses: common.MustOpenTopic(preResponseQueue),
 		jobs:         map[string]*job{},
 	}
+
 	go srv.Receive()
+	conn, err := grpcutil.Dial(fmt.Sprintf("%s:%d", opts.Host, opts.Port), true, opts.CertFile, opts.TokenFile)
+	if err != nil {
+		log.Warning("Failed to connect to bootstrap client: %s", err)
+	}
+	bootstrapClient := bpb.NewBootstrapClient(conn)
+	srv.jobs, err = getExecutions(bootstrapClient)
+	if err != nil {
+		log.Warningf("%s", err)
+	}
+	log.Notice("Updated server with %s inflight executions", len(srv.jobs))
+	conn.Close()
```

Comment:
A good Go practice is to `defer` things like this; after you've opened the connection successfully, `defer conn.Close()`, then it gets cleaned up on any exit.